cactusbot/commands/command.py

```diff
@@ -4,7 +4,8 @@
 import re
 
 ROLES = {
-    5: "Owner",
+    6: "Owner",
+    5: "ChannelEditor",
```

Comment:
We can keep `Owner` as 5 and then just make `ChannelEditor` 4, since it's equivalent to moderator for our generalized purposes.